### ISSUE TITLE
VIVO-1855: i18n : Typos in all_fr_CA.properties

### ISSUE THREAD

**Issue by chenejac** [vivo-project/VIVO#3443]
[Nicolas B Dickn](https://vivo-project.atlassian.net/jira/people/70121:d6633ae6-182e-4293-a5d8-41f16a4519d5) (Migrated from [VIVO-1855](https://vivo-project.atlassian.net/browse/VIVO-1855)) said:

There are still numerous typos left in the following properties files :
 * Vitro-languages\fr_CA\webapp\src\main\webapp\i18n\all_fr_CA.properties
 * VIVO-languages\fr_CA\webapp\src\main\webapp\themes\tenderfoot\i18n\all_fr_CA.properties
 * VIVO-languages\fr_CA\webapp\src\main\webapp\themes\wilma\i18n\all_fr_CA.properties

that could be fixed even without being spotted in the UI.

In other words, the scope of this ticket is to fix typos, grammar and translation errors that are obvious even outside the execution context.

It might eventually overlap with other i18n tickets, which I shall mention here as I find them.

*Note*: This ticket excludes the single quotes, which can cause interpretation problems in Javascript, and must therefore be fixed at two different levels at once.

*Edit:* some single quotes were removed when rephrasing allowed it
